packages/abi-gen-wrappers/src/generated-wrappers/erc1155_proxy.ts

```diff
@@ -31,100 +31,29 @@ import { assert } from '@0x/assert';
 import * as ethers from 'ethers';
 // tslint:enable:no-unused-variable
 
-export type MultiAssetProxyEventArgs =
-    | MultiAssetProxyAuthorizedAddressAddedEventArgs
-    | MultiAssetProxyAuthorizedAddressRemovedEventArgs
-    | MultiAssetProxyAssetProxyRegisteredEventArgs;
+export type ERC1155ProxyEventArgs =
```

Comment:
to be clear, `abi-gen-wrappers` now contains **both** MultiAssetProxyContract and ERC1155ProxyContract. It looks like the file contents are similar enough to confuse git.